config/config.go

```diff
@@ -134,7 +134,18 @@ type Account struct {
 
 var cachedConfigDir string
 var cachedScriptDir string
+var cachedConfigFilePath string
 
+//SetConfigFile sets the configFileName cache to the entered value,
+//bypassing how cordless sets defaults.
+func SetConfigFile(configFilePath string) (string, error) {
+	_, statError := os.Stat(configFilePath)
+	if statError != nil {
```

Comment:
Why does the file have to exist? If it doesn't exist, cordless will simply create it, that's how it usually behaves anyway.